README.md

```diff
@@ -1,70 +1,117 @@
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">SCSS</a></li>
+    <li><a href="https://reactjs.org/">webpack</a></li>
+    <li><a href="https://reactjs.org/">JavaScript</a></li>
+    <li><a href="https://reactjs.org/">ReactJs</a></li>
+    <li><a href="https://reactjs.org/">Redux</a></li>
+  </ul>
+</details>
 
-### `npm test`
+<details>
+<summary>API</summary>
+  <ul>
+    <li><a href="https://www.coingecko.com/en/api">CoinGecko API</a></li>
+  </ul>
+</details>
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Demo
 
-### `npm run build`
+[**See Project 🚀**](https://daxpay.netlify.app)
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Video Demonstration
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+[** Watch 😄 **](https://www.loom.com/share/e55ac30557e442ef98913954f4761434)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Screenshots 
 
-### `npm run eject`
+![Desktop Design](./metric-preview.png)
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## Get Started
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+To get the content of this project locally you need to run the following commands in your terminal.
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+### Prerequisites
+- IDE(code editor) like: **Vscode**, **Sublime**, etc. 
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+### Setup
+1. Download the **Zip** file or clone the repo with:
+```bash
+git clone git@github.com:Strangeal/metric-webapp.git
+```
+3. To access cloned directory run:
+```bash
+cd metric-webapp
+```
+3. Open it with the live server
 
-## Learn More
+### Install
+> To install linters and other project's dependencies run:
+```bash
+npm install
+```
+## Run Test for Linters
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
```

Comment:
- [OPTIONAL] Your README is well documented 💯 I just suggest adding `npm run test` command to run your unit test 👍